stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/service/sync/PushMessageSyncHandler.kt

```diff
@@ -73,31 +79,31 @@ public class PushMessageSyncHandler(private val service: Service) {
             return
         }
         createSyncNotificationChannel()
-        showForegroundNotification(notificationConfig.smallIcon)
+        startForegroundExecution(notificationConfig.smallIcon)
```

Comment:
Are these notification handling methods safe to call from any thread?

Reply:
Yes, the `service.startForeground()` is safe to run from any thread.